onnx_neural_compressor/quantization/matmul_nbits_quantizer.py

```diff
@@ -189,7 +200,7 @@ def int4_quant_algo(self):
             sess_options.add_session_config_entry(
                 "session.optimized_model_external_initializers_min_size_in_bytes", "1024"
             )
-            session = ort.InferenceSession(model, sess_options)
+            session = ort.InferenceSession(model, sess_options, providers=["CPUExecutionProvider"])
```

Comment:
do we only support CPU EP?

Reply:
Here we only want to optimize the graph so use CPUExecutionProvider is enough. For other EPs like TensorRT which contain EP-specific fused pattern, it will crash during optimization